### DISCUSSION TITLE
Having trouble getting Lax-Wendroff to be stable

### DISCUSSION THREAD

**Discussion by jml211crs** [Part-II-Computational-Physics/.github#12] [General]
I've been trying to find the region where Lax-Wendroff is stable. The only way I can get stability is with large values of dx and/or small values of dt. This also means I haven't been able to get the error code working.

I seen that convergence should happen when |v*dt/dx| <= 1, but that's not working for me. The upper bound I've found is more like 0.02.

Any advice would be greatly appreciated!

**Comment by NathanMagnan** [comment]
Yep, that seems like a problem. I would guess that it comes from a typo in your Lax-Wendroff scheme. 
Have you checked if the animation looks reasonable below 0.02 (Is it a simple advection, does it advect at the right speed?) 
Sorry I can't be of more help...

**Reply by jml211crs** [reply]
I changed the method I was using and it works now! I was using list comprehensions rather than np.roll, I think it was something there.
